frontend/public/components/storage-class-form.tsx

```diff
@@ -55,7 +57,111 @@ export class StorageClassForm_ extends React.Component<
     this.previousName = '';
   }
 
-  storageTypes = Object.freeze({
+  defaultProvisionerObj = {
+    title: '',
+    provisioner: '',
+    parameters: {},
+  };
+
+  CSIStorageTypes = {
+    'rbd.csi.ceph.com': {
+      title: 'Ceph RBD',
+      provisioner: 'rbd.csi.ceph.com',
+      documentationLink: 'https://rook.github.io/docs/rook/master/ceph-block.html',
```

Comment:
don't we need to fix this?

Reply:
@madhu We don't want to change the link every time in every release of UI. So is there a stable link I can provide?

Reply:
@travisn can we use https://rook.io/docs/rook/v1.1/ link here?

Reply:
Rook doesn't currently have stable links except master, but you don't want to link to master docs since that will change. There is also a big warning at the top that you're looking at unreleased documentation. So you would need to use the v1.1 links.

Reply:
Actually, does it make sense to link to the upstream documentation? This is for creating an OCS storage class, right? Shouldn't we be linking to downstream docs?

Reply:
@travisn Will the v1.1 link be the same or do we need to change it every time in a release?

IMO, This is an upstream master code so it should not link downstream somewhere.

@spadgett opinion??

Reply:
I'd also say that 1.1 https://rook.io/docs/rook/v1.1/ is better than master. Ideally the link should come from some config map populated by rook operator